### PR TITLE
Merge 17.4 code freeze into develop

### PR DESCRIPTION
**Pull request by jkmassel**
Merges the `17.4` code freeze into `develop`.

There are three changes made as part of this freeze:
- I had to fix an issue with the new secrets format in AppCenter builds
- I've moved the PR list fetching up in the process to avoid issues with the frozen tag
- I've had to temporarily disable Google login due to the new secrets format – they'll be restored this week

### CODE REVIEW

**Comment by loremattei** [comment]
> I've had to temporarily disable Google login due to the new secrets format – they'll be restored this week

Does it have any impact on logged-in users which logged in with Google?
Has this been shared with WPMobile people (I haven't seen any discussion)?

**Comment by jkmassel** [comment]
The Google Login stuff was fixed by https://github.com/wordpress-mobile/WordPress-iOS/pull/16546, so we just need to land this so I can do a followup beta

File: WordPress/Info.plist [hunk 1]
```diff
@@ -447,9 +447,7 @@
 			<key>CFBundleURLName</key>
 			<string>GoogleSignIn</string>
 			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>GOOGLE_LOGIN_SCHEME_ID</string>
-			</array>
```
Comment (oguzkocer) [inline WordPress/Info.plist:452]: @jkmassel I see that Google login issue is fixed, so are these removals still expected?

Comment (jkmassel) [reply WordPress/Info.plist:452]: Yep, they'll land in the next beta, then I'll cut another PR to merge that fix back to `develop`!

**Review by oguzkocer — APPROVED** [review]

**Pull request merged**